components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -1525,7 +1527,7 @@ spec:
           value: tekton-results-api-service.tekton-pipelines.svc.cluster.local:8080
         - name: AUTH_MODE
           value: token
-        image: quay.io/redhat-appstudio/tekton-results-watcher:3429e667f92396aa273eb60c4212105ca2ffda9b
+        image: quay.io/redhat-appstudio/tekton-results-watcher:1c5b3054ffb52f172fda31587d7dfd88a9724c8f
```

Comment:
this is the current level in prod @enarha so this lines up with our intent to keep prod results constant while you vet the nightlies